internal/connect/command_custom_plugin_list.go

```diff
@@ -56,3 +62,8 @@ func (c *customPluginCommand) list(cmd *cobra.Command, _ []string) error {
 	}
 	return list.Print()
 }
+
+func (c *customPluginCommand) addListCloudFlag(cmd *cobra.Command) {
```

Comment:
This flag is not called `--list-cloud`, so let's change this function back to it's original name. If there's a naming conflict with an existing `addCloudFlag()` function, we can consider passing a boolean to change the functionality.